### ISSUE TITLE
It seems that all code examples in docs are not compiling

### ISSUE THREAD

**Issue by vsapronov** [SuaveIO/suave#622]
Check this: https://suave.io/index.html - code sample is not compilable, seems some namespaces changed.
Check this: https://suave.io/routing.html - code sample is not compilable , seems some types have changed.

**Comment by vsapronov** [comment]
I'm closing issue - it seems that I was using Suave 0.30 somehow instead of 2.1

**Comment by StefanBelo** [comment]
I downloaded project trying to understand how suave works, and as you reported I cannot build suave solution at all. To test examples at least I unloaded all projects but left examples, and adding reference to suave assemblies from nugget.

**Comment by StefanBelo** [comment]
Getting 124 errors starting:

open Suave.Logging.Message

Severity	Code	Description	Project	File	Line	Suppression State
Error		The namespace 'Message' is not defined	Suave	G:\Samples\suave-master\src\Suave\Authentication.fs	8

**Comment by haf** [comment]
@StefanBelo You're talking about a different problem than this issue was tracking. How to build is clearly documented in the README and it's not just to open visual studio to get cracking. That said, we have *a lot* of samples are *are* made for making it easy:

 - https://github.com/lefthandedgoat/genit
 - https://github.com/fable-compiler/fable-suave-scaffold
 - https://www.gitbook.com/book/theimowski/suave-music-store/details
 - https://github.com/haf/suave-presentation.2015-09-03

Just mention just a few. Hope these help.

**Comment by StefanBelo** [comment]
Thanks haf, but I think me and vsapronov meant the same.

I downloaded suave source code that includes examples as well. First I tried run build.cmd from developers console, paket restored all dependencies. 

Compilation ended with lots of errors.

Then I repeated the similar on clear source code again using VS (I have hot paket install), so the same like I run build.cmd, VS started building the code ending with errors.

I wonder how I should build suave on windows platform.
